ts_types/DAOVotingAdapterBase.client.ts

```diff
@@ -1,10 +1,10 @@
 /**
-* This file was automatically generated by @cosmwasm/ts-codegen@1.13.3.
```

Comment:
Please upgrade ts-codegen to latest version and regenerate the schema. Changes should be present only in InflowVault TS files.